release/nightly_tests/shuffle/shuffle_10gb.py

```diff
@@ -0,0 +1,14 @@
+import subprocess
+import time
+import json
+
+if __name__ == "__main__":
+    start = time.time()
+    # 10GB shuffle
+    subprocess.check_call([
+        "python", "-m", "ray.experimental.shuffle", "--ray-address=auto",
```

Comment:
Btw @krfricke should we also pass ray address in as a flag? Otherwise you can't run the test locally without changing the code.

Reply:
In this case, shouldn't this work if you start a ray cluster on your local machine?

Maybe I'm missing something here though

Reply:
It would be nice to be able to run any test as `python test.py` without needing to do extra things like that though.

Reply:
It would be nice to be able to run any test as `python test.py` without needing to do extra things like that though.

I think the following would be most flexible:
1. Pass RAY_ADDRESS, TEST_OUTPUT_JSON, and IS_SMOKE_TEST as env vars.
2. Allow the test script to take arguments as well.

This would allow the most flexibility for test script writers. For example, for shuffle tests we could define just 1 script, and have a whole bunch of variations just defined in the nightly_test.yaml, instead of 1 script per test configuration.

Reply:
Ok, that sounds like a good idea.

For arguments, should we just specify an `args` key in the `run` block?

Reply:
Sounds good. I think either would work:
```
script: shuffle/shuffle_10gb.py --some --args
```
or
```
script: shuffle/shuffle_10gb.py
args: ["--some", "--args"]
```

Reply:
Ok, got it. The first option already works. I'll add the second option, too (it will just append `" ".join(args)` to the script)

Reply:
I'll get all of this done tomorrow morning

Reply:
Ah didn't know that. I'd be happy with just the first then!

Reply:
Added!